src/platform/testing/unit/mocha-setup.js

```diff
@@ -30,8 +36,21 @@ export default function setupJSDom() {
 
   // Prevent warnings from displaying
   /* eslint-disable no-console */
-  console.error = () => {};
-  console.warn = () => {};
+  if (process.env.LOG_LEVEL === 'debug') {
+    console.error = (error, reactError) => {
+      if (reactError instanceof Error) {
+        console.log(filterStackTrace(reactError.stack));
+      } else if (error instanceof Response) {
+        console.log(`Error ${error.status}: ${error.url}`);
+      } else if (error instanceof Error) {
+        console.log(filterStackTrace(error.stack));
+      }
+    };
+    console.warn = () => {};
+  } else if (process.env.LOG_LEVEL !== 'trace') {
```

Comment:
If you use 'trace', then all the output in its original form is returned.

Reply:
I think this is a useful addition. Displaying all the output may be desired when debugging extensively.